lib/akin.ex

```diff
@@ -52,17 +52,14 @@ defmodule Akin do
 
   def compare(%Corpus{} = left, %Corpus{} = right, opts) do
     Enum.reduce(list_algorithms(opts), %{}, fn algorithm, acc ->
-      Map.put(acc, algorithm, apply(modulize(algorithm), :compare, [left, right, opts]))
-    end)
-    |> Enum.reduce([], fn {k, v}, acc ->
-      if is_nil(v) do
+      score = apply(modulize(algorithm), :compare, [left, right, opts])
+
+      if is_nil(score) do
         acc
       else
-        [{String.replace(k, ".", ""), v} | acc]
```

Comment:
I couldn't find any alg with "." in its name. 
Should I keep this replace?

Reply:
I'm taking a look at some other areas and will remove this as part of that effort. Thank you!

Reply:
Verified this code is needed; it is modifying floats, not algorithm names.